src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/domene/GrunnlagsdataDomene.kt

```diff
@@ -66,11 +68,18 @@ data class GrunnlagsdataDomene(
     val barn: List<BarnMedIdent>,
     val tidligereVedtaksperioder: TidligereVedtaksperioder?,
     val harAvsluttetArbeidsforhold: Boolean?,
-    val harKontantstøttePerioder: Boolean?,
+    val kontantstøttePerioder: List<KontantstøttePeriode> = emptyList(),
```

Comment:
Nå er vi ikke bakoverkompatibel... Selv om vi har hatt "true" på `harKontantstøttePerioder` i tidligere (og gjeldende) behandlinger så vil det blir `false` fremover. 

Er dette avklart med Lars og Mirja? Det kan hende vi må tenke litt mer "expand-contract" og bakoverkompabilitet rundt denne overgangen. Det gjelder vel også frontend og blankett hvis vi trenger å vise historiske verdier her.

Reply:
Godt poeng! Tror vi ble litt ivrige med å fjerne dette flagget. Legger det tilbake, og så kan vi satse på å ha det som en "expand" frem til det er mulig å fjerne det ?

Reply:
Er nok aldri mulig å fjerne harKontantstøttePerioder. Vil alltid gjelde for gamle behandlinger.